docs/en/writing-running-appium/ios/ios-xctest-performance.md

```diff
@@ -0,0 +1,52 @@
+## Automating Performance Metrics Measurement
+
+Apple provides `instruments` tool with quite rich set of features for desktop and mobile applications performance measurement. The collected data can be then visualized with `Instruments.app`, which is a part of Xcode DevTools. By default Xcode provides several measurement templates, like `Activity Monitor` or `Time Profiler`, but one can also create own profiles and select custom set of metrics to record and visualize. Read https://developer.apple.com/library/content/documentation/DeveloperTools/Conceptual/InstrumentsUserGuide to get more detailed overwiew of available tool features.
```

Comment:
`one can also create their own profiles and select a custom set`

`Read ... to get a more detailed overview`